bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -696,26 +696,73 @@ sitemap netatmo label="Netatmo" {
 }
 ```
 
+## Rule Actions
 
-# Sample data
+Multiple actions are supported by this binding. In classic rules these are accessible as shown in this example (adjust getActions with your ThingId):
+
+Example
+
+```
+ val actions = getActions("netatmo","netatmo:room:home:livingroom")
+ if(null === actions) {
+        logInfo("actions", "Actions not found, check thing ID")
+        return
+ }
+```
+
+### setThermRoomTempSetpoint(temp,endtime)
+
+Sends a temperature setpoint (and switch to manual mode) to the thermostat for a room with an end time.
+
+Parameters:
+
+| Name    | Description                                                          |
+|---------|----------------------------------------------------------------------|
+| temp    | The temperature setpoint                                             |
+| endtime | Time the setpoint should end                                         |
+
+Example:
+
+```
+actions.setThermRoomTempSetpoint(19.0,240)
```

Comment:
@clinique : please explain what is the unit of the endtime. Is it a duration in seconds ?

Reply:
Enddate of the therm mode (Local Unix Time in seconds).

Reply:
This is not top !
Do you know what would be the syntax in DSL rule to create this parameter ?

Reply:
Agree, this should better handle a DateTimeType so compatibility with scripting will be easiest. I will take a look at this in a separate PR if you agree.

Reply:
Description updated.